scalacheck-effect/src/test/scala/org/specs2/scalacheck/effect/ScalaCheckEffectMatchersApiSpec.scala

```diff
@@ -0,0 +1,105 @@
+package org.specs2
+package scalacheck.effect
+
+import _root_.cats.effect.IO
+import org.scalacheck.Prop.*
+import org.scalacheck.effect.PropF.*
+import org.scalacheck.Prop.propBoolean
+import org.scalacheck.util.{FreqMap, Pretty}
+import org.scalacheck.*
+import org.specs2.cats.effect.IOExecution
+import org.specs2.Specification
+
+class ScalaCheckEffectMatchersApiSpec extends Specification, ScalaCheckEffect, IOExecution:
+  def is = s2"""
+ There are various ways to use a property in specs2
+
+ A ScalaCheck property can be used in the body of an Example
+ ${forAllF { (i: Int) => IO(i > 0 || i <= 0) }}
+
+ The `prop` method can be used to create a property from a function
+    returning a result
+    ${propF { (i: Int) => IO(success) }}
+    returning a match result
+    ${propF { (i: Int) => IO(i must (be_>(0) or be_<=(0))) }}
+    returning a boolean value
+    ${propF { (i: Int) => IO(i > 0 || i <= 0) }}
+    using  an implication and a match result
+
+    // propBoolean is used - can be removed when scalacheck 1.15.0 is out
+    {prop { (i: Int) => propBoolean(i > 0) ==> (i must be_>(0)) }}
```

Comment:
Maybe it's worth trying to remove `propBoolean` now

Reply:
Sounds good, I'll remove these from here then :)